library/pluginsystem/PluginSystem.php

```diff
@@ -76,7 +76,7 @@ public function init()
         $composerJsonPath = realpath( __DIR__."/../../composer.json" );
         $composerJson = file_get_contents( $composerJsonPath );
         if ( $composerJson === false ) {
-            throw new Exception( "$composerJsonPath/composer.json is not readable" );
+            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
```

Comment:
Ken,

Though it's not a critical or even a issue, the error is wrong.

This:

>            throw new \Exception( "$composerJsonPath/composer.json is not
readable" );

Should be:

>            throw new \Exception( "$composerJsonPath is not readable" );

The filename is already in the string.
Sam

On Sep 5, 2016 1:17 PM, "Ken" <notifications@github.com> wrote:

> In library/pluginsystem/PluginSystem.php
> <https://github.com/LibreEHR/LibreEHR/pull/151#discussion_r77546057>:
>
> > @@ -76,7 +76,7 @@ public function init()
> >          $composerJsonPath = realpath( __DIR__."/../../composer.json" );
> >          $composerJson = file_get_contents( $composerJsonPath );
> >          if ( $composerJson === false ) {
> > -            throw new Exception( "$composerJsonPath/composer.json is not readable" );
> > +            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
>
> Yes. The plugin that adds the video conferencing functionality is on my
> github right now. https://github.com/kchapple/vidyo
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreEHR/LibreEHR/pull/151/files/2adaccb39c50060f8df9c4a63439d7020a2235bd#r77546057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdEYIA7Id5sWGlZwHLdTvttoedRe8UNks5qnE6MgaJpZM4J1MvO>
> .
>